src/sentry/models/group.py

```diff
@@ -175,18 +175,20 @@ def from_event_id(self, project, event_id):
         return Group.objects.get(id=group_id)
 
     def filter_by_event_id(self, project_ids, event_id):
-        from sentry.models import EventMapping, Event
-        group_ids = set()
-        # see above for explanation as to why we're
-        # looking at both Event and EventMapping
-        for model in Event, EventMapping:
-            group_ids.update(
-                model.objects.filter(
-                    project_id__in=project_ids,
-                    event_id=event_id,
-                    group_id__isnull=False,
-                ).values_list('group_id', flat=True)
-            )
+        from sentry.utils import snuba
+
+        group_ids = set([evt['issue'] for evt in snuba.raw_query(
+            start=datetime.utcfromtimestamp(0),
```

Comment:
Is the retention policy we have in snuba the same as the one we have in Event and EventMapping table? Snuba will apply can apply a default maximum timespan.
https://github.com/getsentry/snuba/blob/master/snuba/api.py#L170

@JTCunning Do we use that setting in production?

Reply:
We do have this setting in Production, but this should be scoped to an organization's retention.

You can find other queries scoped to retention like so: https://github.com/getsentry/sentry/blob/0a8c456a4efd1a162b5eb87d09a1ddab6be8a631/src/sentry/search/snuba/backend.py#L275-L285

This also has lead me to believe that we have most likely not properly scoped other SnubaEvent-based queries to organization retention.

Reply:
As discussed on Slack, Sentry's snuba client shrinks every time window to be inside retention. https://github.com/getsentry/sentry/blob/576048a7500e7fc79d036bcb0cc7b66df495b1ad/src/sentry/utils/snuba.py#L561-L574

Reply:
Which goes back to my original question. 
Are we going to return fewer results than in the previous implementation since there is no retention applied to the postgres query ?

Reply:
No. Those are deleted at 90d every 5 minutes, and if we weren't previously capping the 30d retention, now is the time to.

Reply:
@JTCunning Looks like this is already being handled for all Snuba queries https://github.com/getsentry/sentry/blob/master/src/sentry/utils/snuba.py#L563-L585